src/software/ai/hl/stp/tactic/chip/chip_fsm.h

```diff
@@ -69,7 +69,7 @@ struct ChipFSM
         };
 
         return make_transition_table(
-            // src_state + event [guard] / action = dest state
+            // src_state + event [guard] / action = dest_state
             *get_behind_ball_s + update_e / update_get_behind_ball,
```

Comment:
does the * indicate the initial state?

Reply:
yes, whoops I'll add that to the documentation